### ISSUE TITLE
RaspAP on OMV6 without dhcpd

### ISSUE THREAD

**Issue by piter-pit** [RaspAP/raspap-webgui#1179]
**Is your feature request related to a problem? Please describe.**
I was trying to set up AP on OMV6 however neither RaspAP nor manual setup of hostapd was successful due to OMV uses systemd-networkd and removes dhcpcd during installation.
I've tried installation of manual installation of
`sudo apt install dhcpcd5`
and then RaspAP and manual setting but without success

**Describe the solution you'd like**
Would it be possible to fix this case?

**Describe alternatives you've considered**

**Additional context**
Subject was also raised on [omv-docs](https://github.com/openmediavault/openmediavault-docs/issues/102)

**Comment by billz** [comment]
Not really. `systemd-networkd` is fundamentally incompatible with this project and all dhcpcd / dnsmasq based AP configurations.

Per the FAQ the solution is to skip OMV's network setup: https://docs.raspap.com/faq/#omv

The same is also [advised](https://forum.openmediavault.org/index.php?thread/43315-omv6-as-access-point/&postID=314081#post314081) in the OMV forums. There is no other workaround.
